modules/virt-vm-migration-tuning.adoc

```diff
@@ -0,0 +1,27 @@
+
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-about-live-migration.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vm-migration-tuning_{context}"]
+= VM migration tuning
+
+Live migration requires the use of a shared storage solution that provides `ReadWriteMany` (RWX) access mode. VM disks must be backed by volumes defined with RWX.
+
+{VirtProductName} checks whether a VM can be live-migrated. If a VM can be live-migrated, the `evictionStrategy` is set to `LiveMigrate`.
+//where does the user set the eviction strategy?
```

Comment:
Question 6

Reply:
@jean-edouard  I took this from the Kbase article. Should I remove it? IS there something I should be telling the user instead?

Reply:
Yeah I'd remove it, it's not technically a migration setting anyway.